### DISCUSSION TITLE
How to use BEITv2 in version dev1.x?

### DISCUSSION THREAD

**Discussion by yfzhang114** [open-mmlab/mmselfsup#675] [How-to]
I try to use the pretrained BEITv2 model, but the target generator has a checkpoint beit_ckpt/vqkd_encoder.pth, which is not provided. To address the problem, I download the checkpoint from https://github.com/microsoft/unilm/blob/master/beit2/modeling_vqkd.py, however, the params are not the same.

**Comment by fangyixiao18** [comment]
Sorry for the inconvenience, we forgot to update the vqkd_encoder.pth in config, I will create a PR to update it. You could download our file, we have align the params keys. 
https://download.openmmlab.com/mmselfsup/1.x/target_generator_ckpt/vqkd_encoder.pth

**Comment by yfzhang114** [comment]
Many thanks!

Yixiao Fang ***@***.***> 于2023年1月16日周一 16:39写道：

> Sorry for the inconvenient, we forgot to update the vqkd_encoder.pth in
> config, I will create a PR to update it. You could download our file, we
> have align the params keys.
>
> https://download.openmmlab.com/mmselfsup/1.x/target_generator_ckpt/vqkd_encoder.pth
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/open-mmlab/mmselfsup/discussions/675#discussioncomment-4694904>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AMVS7XVHFGRT7LWQXP7EXKDWSUCLRANCNFSM6AAAAAAT4GJJYY>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>


-- 
Yifan Zhang
South China University of Technology, China
Email: ***@***.***
